ui/store/actions.ts

```diff
@@ -4851,9 +4852,41 @@ export function getGasFeeTimeEstimate(
   ]);
 }
 
-export async function closeNotificationPopup() {
+export async function attemptCloseNotificationPopup() {
+  // Check if the current window is NOT a popup - if confirmed, we should not close it
+  try {
+    const currentWindow = await browser.windows.getCurrent();
+    if (currentWindow.type && currentWindow.type !== 'popup') {
```

Comment:
Nit: `browser.windows.WindowType.popup` returns `"popup"`, if you're into using that sort of thing. haha

Reply:
Oh, typo in the casing there.

Interesting that the types and docs don't match up. I'm good with using the string here.